hastory/src/Hastory/Storage/HastoryZonedTime.hs

```diff
@@ -0,0 +1,17 @@
+module Hastory.Storage.HastoryZonedTime where
+
+import Prelude ((.), undefined)
+import Data.Time (ZonedTime, zonedTimeToUTC)
+import Database.Persist (PersistField(..), SqlType(..))
+import Database.Persist.Sql (PersistFieldSql(..))
+
+newtype HastoryZonedTime = HastoryZonedTime { unHastoryZonedTime ::  ZonedTime }
+
+instance PersistField HastoryZonedTime where
+    toPersistValue = toPersistValue . zonedTimeToUTC . unHastoryZonedTime
+
+    -- fromPersistValue :: PersistValue -> Either Text a
+    fromPersistValue persistValue = undefined
```

Comment:
Yeah, this is tricky, isn't it.
I think we can get away with just using `UTCTime` instead of `ZonedTime` instead of messing with a `ZonedTime` here.
Alternatively we could store the `LocalTime` and `TimeZone` seperately in the database.

Reply:
`UTCTime` it is!